package.json

```diff
@@ -72,19 +82,23 @@
     "vm-browserify": "^1.0.0",
     "xtend": "^4.0.0"
   },
+  "//devDependencies": {
+    "//": "Comments about why some devDependencies are at specific versions that may issue warnings.",
+    "rimraf@2.6.3": "Dependency of temp. No update for temp available."
+  },
   "devDependencies": {
     "backbone": "~0.9.2",
     "browser-unpack": "^1.1.1",
-    "coffee-script": "~1.10.0",
-    "coffeeify": "~1.1.0",
+    "coffeescript": "^2.7.0",
+    "coffeeify": "^3.0.1",
     "has-object-spread": "^1.0.0",
     "has-template-literals": "^1.0.0",
     "isstream": "^0.1.2",
     "make-generator-function": "^1.1.0",
     "semver": "^5.5.0",
     "seq": "0.3.5",
-    "tap": "^10.7.2",
-    "temp": "=0.8.3",
+    "tap": "^21.0.1",
```

Comment:
this version of tap requires node 20 or 22+, so it's not an option unless browserify requires it as well.

Reply:
Gotcha. Is there a specific baseline version of Node you'd prefer as the minimum version for developing Browserify vs minimum version to use it? Or 0.8 for both?

Reply:
To avoid a breaking change, we need to stick to 0.8.

Development isn't the issue, it's testing - CI needs to be able to run in all supported node versions.

Reply:
Gotcha, makes sense. I saw in some previous PRs you were hesitant to drop support for some earlier versions of Node. Is there a point at which you'd reconsider? I saw some npm audit findings of high and critical severity when I first cloned. I know v17 is pretty fresh already and it might be annoying to do another major version so soon, but that would be something to think about.

Reply:
I think anything like this will have to be preceded by this: #2071
Would be fantastic if it would be taken up for a fresh look and reconsideration by maintainers.

Reply:
Most audit warnings don't apply, or are false positives - that's not much of a motivator.

Reply:
> I think anything like this will have to be preceded by this: #2071
> 
> Would be fantastic if it would be taken up for a fresh look and reconsideration by maintainers.

I wasn't sure if that stance applied only to direct dependencies or to dev dependencies as well, hence the above conversation.

It's not a huge deal for me -- the places where I use it, I use it with Yarn and am comfortable resolving deep dependencies to newer versions if the infosec at my workplace requires it. I just tend to apply that mindset to everything, regardless of actual impact of potential vulnerabilities, but I respect the maintainer's wishes here even if it's not the approach I'd personally take.

Reply:
Apps and packages demand very different stances and philosophies around back compat and hacks, as well as security warnings.